syntax = "proto3";

package example;

service MathService {
  rpc Add (AddRequest) returns (AddResponse);
  rpc Multiply (MultiplyRequest) returns (MultiplyResponse);
  
}

message AddRequest {
  float a = 1;
  float b = 2;
}

message AddResponse {
  float result = 1;
}

message MultiplyRequest {
  float a = 1;
  float b = 2;
}

message MultiplyResponse {
  float result = 1;
}
